Sources/SwiftCBOR/CBOREncoder.swift

```diff
@@ -1,336 +1,914 @@
-#if canImport(Foundation)
+//
+//  CBOREncoder.swift
+//  Sunday
+//
+//  Created by Kevin Wooten on 7/11/18.
+//  Copyright © 2018 Outfox, Inc. All rights reserved.
```

Comment:
Don't use "all rights reserved", it's not necessary. Especially when that's not the case (with any FOSS licenses) :)